src/ui/Embed.tsx

```diff
@@ -60,5 +60,6 @@ function resolveYoutube(videoId: string): JSX.Element {
 }
 
 function resolveTwitch(videoId: string): JSX.Element {
-    return videoIframe(`https://player.twitch.tv/?video=${videoId}&autoplay=false`);
+    const domain = process.env.NODE_ENV === "production" ? "one.livesplit.org" : "localhost";
```

Comment:
In the thread I linked the dynamically get the domain from the browser. Maybe that makes more sense than hardcoding the values.

Reply:
Seems like `window.location.hostname` works:

![chrome_JCWsWMx0qg](https://user-images.githubusercontent.com/8262173/84449320-ac665d80-ac12-11ea-8e73-39228286f569.png)
![chrome_YWuXNR5PP6](https://user-images.githubusercontent.com/8262173/84449321-ac665d80-ac12-11ea-8ea0-de720647c9d4.png)